Added e2e tests for `page.tag.detached` webhook

Dark-Knight11 opened the pull request:
refs: #15537

This adds an e2e test and test snapshot for the `page.tag.detached` webhook so we can prevent regressions and bugs in the future

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

Also, if you'd be interested in writing code like this for us more regularly, we're hiring:
https://careers.ghost.org

naz commented on ghost/core/test/e2e-webhooks/pages.test.js:
```diff
@@ -56,13 +68,47 @@ const buildPageSnapshotWithTiers = ({
     };
 };
 
+const buildPageSnapshotWithTiersAndTags = ({
```
Would it make sense to reuse the `buildPageSnapshotWithTiers` method here?

naz commented on ghost/core/test/e2e-webhooks/pages.test.js:
```diff
@@ -12,9 +12,21 @@ const {
+const tagSnapshot = {
+    created_at: anyISODateTime,
+    id: anyObjectId,
+    name: anyString,
```
The tag's name, slug, visibility etc. are not changing between test runs? If they are not, would be best to have snapshot capture them as is - no need to include non-dynamic properties in the snapshot matcher object 👍

naz reviewed (changes requested):
Hey-hey @Dark-Knight11 👋  These are great changes are I think we are 90% there. I've left couple of comments around areas that need improvement. Feel free to reach out if you need more guidance and thanks for the contribution!

Dark-Knight11 commented:
> Hey-hey @Dark-Knight11 👋 These are great changes are I think we are 90% there. I've left couple of comments around areas that need improvement. Feel free to reach out if you need more guidance and thanks for the contribution!

Hi, I have made the requested changes kindly look into it and lmk if anymore changes are needed from my side.
Thanks!

Dark-Knight11 committed:
Added e2e tests for page.published webhook

ref #15537

- this adds an e2e test and test snapshot for the page.published webhook
so we can prevent regressions and bugs in the future

naz commented:
The snapshots seem to be outdated on and are failing now. Can you please re-record them again @Dark-Knight11 🙏

naz reviewed (approved):
Great work on these changes @Dark-Knight11! Looking forward for more contributions 😊

Dark-Knight11 committed:
Delete config.development.json

ErisDS committed:
Updated snapshots

Dark-Knight11 committed:
Added e2e test for page.tag.detached event

refs: TryGhost#15537

This adds an e2e test and test snapshot for the page.tag.detached webhook
so we can prevent regressions and bugs in the future

ErisDS commented:
Hey @Dark-Knight11 thanks again for your contributions! This has been merged and the `hacktoberfest-accepted` label added to make sure it counts.

Please keep them coming 🙏 we have so much we want to achieve beyond hacktoberfest 😬
